java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -201,4 +203,142 @@ public CompletableFuture<String> set(String key, String value, SetOptions option
                         .build();
         return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
     }
+
+    /**
+     * Decrements the number stored at `key` by one. If `key` does not exist, it is set to 0 before
+     * performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/decr/">redis.io</a> for details.
+     * @param key - The key to decrement its value.
```

Comment:
```suggestion
     * @param key The key to decrement its value.
```

Reply:
Remove the dashes everywhere - they are added by doc generator

![image](https://github.com/Bit-Quill/glide-for-redis/assets/88679692/a8efbbb6-0150-49a8-afad-eee436181134)